tests/models/gpt_neox/test_modeling_gpt_neox.py

```diff
@@ -396,6 +396,68 @@ def test_model_rope_scaling(self):
             torch.testing.assert_close(ntk_sin_long, original_sin_long)
         self.assertTrue((ntk_scaling_rope.inv_freq <= original_rope.inv_freq).all())
 
+    @require_torch_sdpa
+    @slow
+    def test_eager_matches_sdpa_generate(self):
```

Comment:
Nit: Added under the `GPTNeoXModelTest` test class but might fit better under `GPTNeoXLanguageGenerationTest`?

Reply:
Closing it. I think I confused some things.